docs/content/documentation/getting-started/installation.md

```diff
@@ -77,6 +77,21 @@ Zola is available in the official package repository.
 $ doas pkg_add zola
 ```
 
+### pkgsrc
+
+Zola is available in the official package repository, with [pkgin](https://pkgin.net/),
+pkg_add\(1\) or [other ways](https://www.netbsd.org/docs/pkgsrc/pkgsrc.html).
+
+```sh
+$ pkgin install zola
+```
+
+or
+
+```sh
+$ pkg_add zola
+```
```

Comment:
Is there any difference between the 2 commands?

Reply:
Yes. But then again for various reasons it makes sense to just mention pkgin here.

Sidenote: it might take a little bit longer for the current version of zola to be available, minor mistake on my side.

Reply:
agreed, that makes more sense.
as I'm occupied with other things, this could take a while. would you prefer that I close this merge request and simply sent a new one once we have an updated binary?

Reply:
Up to you, I don't mind keeping this one up

Reply:
took a while, forgot about this, and I don't know for sure if the current version of zola is already available on the CDN for pkgsrc, but it should catch up soon if it isn't.